kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/asset/Asset.java

```diff
@@ -69,7 +67,7 @@
      * @return the list of asset records which comprises the currently read
      *         value in case of success or the reason of failure
      */
-    public List<AssetRecord> read(List<Long> channelIds) throws KuraException;
+    public List<AssetRecord> read(Set<String> channelNames) throws KuraException;
```

Comment:
Are we considering case sensitivity? According to the contract of string equality, two strings with different cases are considered to be different strings and the set can contain two same strings with different cases.